openspec/changes/archive/2026-02-10-navbar-brand-icon-update/proposal.md

```
@@ -0,0 +1,220 @@
+    alt="Hagicode"
+    width="32"
+    height="32"
+  />
+</div>
+```
+
+### Logo 资源来源
+
+正式 Logo 资源位于 **Desktop 仓库** (`/home/newbe36524/repos/newbe36524/desktop`)：
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)